### ISSUE TITLE
Suppress Build Warnings

### ISSUE THREAD

**Issue by cubriq** [microsoft/azuredatastudio#17900]
Not working:

`  <PropertyGroup>`
`    <NoWarn>$(NoWarn);SQL71558</NoWarn>`
`  </PropertyGroup>`

### Discussed in https://github.com/microsoft/azuredatastudio/discussions/15148

<div type='discussions-op-text'>

<sup>Originally posted by **memjr** April 15, 2021</sup>
Hello, quick question.  We started using Azure Data Studio to work with database projects and when building we get build warnings, more specifically warnings that we don't care about at this time, such as SQL71558 "reference differs only by case from object difinition".

I've edited our .sqlproj file manually and added the bit in bold shown below.  While this works, I wanted to ask if this is good practice or if there is a more appropriate way to handle this (other than changing the code, which we eventually will).

`<?xml version="1.0" encoding="utf-8"?>`
`<Project.......>`
`.....`
`  <PropertyGroup>`
`    <NoWarn>$(NoWarn);SQL71558</NoWarn>`
`  </PropertyGroup>`
`.....`
`</Project>`</div>

**Comment by kisantia** [comment]
@cubriq where in the sqlproj are you adding this property group? I was able to repro this not working when I added the NoWarn property group below the ItemGroups with the table and view of my project. When I moved the NoWarn up to below the other PropertyGroup with the collation etc, then build succeeded without any warnings, so looks like the location that it gets added in the sqlproj matters.

**Comment by philosophicles** [comment]
For the benefit of any others finding this via Google: I've just banged my head against the wall trying to suppress error 71558 for an hour or so. The syntax above was not working for me no matter where I placed it in my sqlproj file.

Eventually, I used full Visual Studio 2022 (with SSDT extension) to open the sqlproj and use the project properties GUI to key in 71558 in the field for suppressed warning numbers. 

The result in my .sqlproj file was as follows

```
<PropertyGroup>
    <SuppressTSqlWarnings>71558</SuppressTSqlWarnings>
</PropertyGroup>
```

and this was added as a separate <PropertyGroup> inside the <Project>. As per @kisantia says above, the new <PropertyGroup> was added immediately after the pre-existing <PropertyGroup> where the name, project GUID, model collation, etc are set, and before the first <ItemGroup> block.

This syntax worked for me to actually suppress the warning when building the project from Azure Data Studio. I'm on SDK version 0.1.14-preview, and version 1.4.1 of the SQL Database Projects extension to ADS.
